packages/fast-navigation-generator-react/webpack.config.js

```diff
@@ -0,0 +1,45 @@
+const path = require("path");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const appDir = path.resolve(__dirname, "./app");
+const outDir = path.resolve(__dirname, "./www");
+
+module.exports = {
+    entry: path.resolve(appDir, "index.tsx"),
```

Comment:
Can you include `devTool: process.env.NODE_ENV === "production" ? "none" : "inline-source-map"` so we can debug if necessary?